cmd/backup-manager/app/util/util.go

```diff
@@ -421,3 +422,21 @@ func ConstructRcloneArgs(conf string, opts []string, command, source, dest strin
 	}
 	return args
 }
+
+// GetContextForSignal get a context for some signals, and the context will become done after any of these signals triggered.
+func GetContextForSignal(op string) (context.Context, context.CancelFunc) {
+	ctx, cancel := context.WithCancel(context.Background())
+
+	sc := make(chan os.Signal, 1)
+	signal.Notify(sc,
```

Comment:
should we also listen for `SIGABRT` and `SIGIOT`?

Reply:
many projects in the PingCAP org listen these four signals (like https://github.com/pingcap/tidb/blob/eb81a67b33e763b8ba94f8176b77ce70d931f7a5/util/signal/signal_posix.go#L46-L49), so I just comply with them.

Reply:
ok